opty/direct_collocation.py

```diff
@@ -188,7 +201,24 @@ def __init__(self, equations_of_motion, state_symbols,
             to a directory here.
 
         """
-        self.eom = equations_of_motion
+        Lamda_Matrix = sym.Matrix([[0,0,0,0], [0,0,0,0], [0,0,Lamda,0],[0,0,0,Lamda]])
+	    """
+	    define matrix Lamda =   [0 0 0     0    ]
+			            [0 0 0     0    ]
+ 			            [0 0 lamda 0    ]
+			            [0 0 0     lamda]
+	    """
```

Comment:
I try to follow the Python PEP8 style guide: https://www.python.org/dev/peps/pep-0008/ for all the code. Please read over that to help with consistent formatting.

Also, you can setup your text editor or IDE to automatically detect/fix these kinds of things.